MOBILE_CLIENT/ANDROID/_LOW/APK_MIN_SDK_VERSION/description.md

```diff
@@ -0,0 +1,2 @@
+The `android:minSdkVersion` attribute specifies the minimum Android API level required by the application. Setting a low `minSdkVersion` may allow the application to run on older Android versions but could expose users to security vulnerabilities.
```

Comment:
Here is a link with the deprecation notice:
https://support.google.com/googleplay/android-developer/answer/11926878?hl=en